tests/docker_noinit/Dockerfile

```
@@ -37,6 +37,9 @@ RUN apk add --no-cache curl git bash && \
     cd / && \
     rm -rf /tmp/act-installation
 
+# Install mise for task discovery and execution tests
+RUN curl -fsSL https://mise.run | MISE_INSTALL_PATH=/usr/local/bin/mise sh
```

Hadolint (2.15.1)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)